### PR TITLE
[13.0][IMP] mass_mailing_partner: Add partner_name in error constrain message

### PR DESCRIPTION
**Pull request by victoralmau**
Add partner_name in error constrain message

Related to https://github.com/OCA/social/pull/646

Please, @pedrobaeza and @chienandalu review it.

@Tecnativa TT27250

### CODE REVIEW

File: mail_outbound_static/i18n/es.po [hunk 1]
```diff
@@ -19,11 +20,11 @@ msgstr ""
...
 msgid ""
-"%s is not a valid domain. Please define a list of valid domains separated by"
-" comma"
+"%s is not a valid domain. Please define a list of valid domains separated by "
+"comma"
 msgstr ""
-"%s no es un dominio válido. Por favor defina una lista de dominios validos separados por"
-" comas"
+"%s no es un dominio válido. Por favor defina una lista de dominios validos "
+"separados por comas"
```
Comment (pedrobaeza) [inline mail_outbound_static/i18n/es.po]: Why this change?

File: mass_mailing_partner/models/mailing_contact.py [hunk 2]
```diff
@@ -29,7 +29,8 @@ def _check_partner_id_list_ids(self):
                 )
                 if contact.list_ids & other_contact.mapped("list_ids"):
                     raise ValidationError(
-                        _("Partner already exists in one of " "these mailing lists")
+                        _("Partner already exists in one of these " "mailing lists")
```
Comment (pedrobaeza) [inline mass_mailing_partner/models/mailing_contact.py]: Remove middle quotes

File: mail_outbound_static/i18n/es.po (hunk 1 above)
Comment (victoralmau) [reply mail_outbound_static/i18n/es.po]: Sorry, `--amend` git command cause it.

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Let's merge it as well as in v12:

/ocabot merge patch
